mergify_engine/actions/merge_base.py

```diff
@@ -102,6 +102,8 @@ def strict_merge_parameter(v):
 async def get_rule_checks_status(
     ctxt: context.Context,
     rule: typing.Union["rules.EvaluatedRule", "rules.EvaluatedQueueRule"],
+    *,
+    unrelated_as_failure: bool = True,
```

Comment:
This name is not obvious to understand. Is there a better name or some docstring to help?

Reply:
I changed it